publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/config/PropertiesConfig.java

```diff
@@ -19,6 +19,8 @@
     public static final String MESSAGE_BUS_PORT = "com.ericsson.eiffel.remrem.publish.messagebus.port";
     public static final String VIRTUAL_HOST = "com.ericsson.eiffel.remrem.publish.messagebus.virtualhost";
     public static final String CHANNELS_COUNT = "com.ericsson.eiffel.remrem.publish.messagebus.channels";
+    public static final String USERNAME = "com.ericsson.eiffel.remrem.publish.messagebus.username";
```

Comment:
Why not change to the new URI format, so applications it self don't need to handle all configuration variables etc.
Most of the configuration is handle in the URI field:
public static final String RABBITMQ_URI"amqps://username:password@address/vhost?query-configuration1&query-configuration2&...."

Then application don't need to have variable for every amqp configuration setting, since most of the configuration is provided as config query to the URI 
https://www.rabbitmq.com/uri-spec.html

Here is the GitHub issue link:
https://github.com/eiffel-community/eiffel-remrem-publish/issues/212

Those configuration that need to be kept as is, since that is per applications needs:
EXCHANGE_NAME
DOMAIN_ID
...
...